gulpfile.js

```diff
@@ -0,0 +1,194 @@
+var { execSync } = require('child_process');
+var { writeFileSync, readdirSync, readFileSync } = require('fs');
+
+var glob = require('glob');
+var _ = require('underscore');
+
+var { src, dest, series, watch } = require('gulp');
+var log = require('fancy-log');
+var gHash = require('gulp-hash');
+var gClean = require('gulp-clean');
+var gTerser = require('gulp-terser');
+var gJasmine = require('gulp-jasmine');
+var { SpecReporter } = require('jasmine-spec-reporter');
+var gJshint = require('gulp-jshint');
+
+var source = require('vinyl-source-stream');
+var buffer = require('vinyl-buffer');
+var browserify = require('browserify');
+var babelify = require('babelify');
+
+_.templateSettings.interpolate = /\{\{(.+?)\}\}/g;
+_.templateSettings.escape = /\{\{\{(.*?)\}\}\}/g;
+_.templateSettings.evaluate = /\{\{-(.*?)\}\}/g;
+
+var prodDependencies = [
+  '<script src="https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.1/es5-shim.min.js"></script>',
+  '<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.8.0/jquery.min.js"></script>',
+  '<script src="https://cdnjs.cloudflare.com/ajax/libs/raphael/2.1.0/raphael-min.js"></script>'
+];
+
+var devDependencies = [
+  '<script src="lib/jquery-1.8.0.min.js"></script>',
+  '<script src="lib/raphael-min.js"></script>',
+  '<script src="lib/es5-shim.min.js"></script>'
+];
+
+// precompile for speed
+var indexFile = readFileSync('src/template.index.html').toString();
+var indexTemplate = _.template(indexFile);
+
+var compliments = [
+  'Wow peter great work!',
```

Comment:
you could have totally added your own name here :P this was back when I was proud of myself for setting this up